google-cloud-storage/lib/google/cloud/storage/project.rb

```diff
@@ -384,6 +385,73 @@ def create_bucket bucket_name, acl: nil, default_acl: nil,
           Bucket.from_gapi gapi, service, user_project: user_project
         end
 
+        ##
+        # Creates a new HMAC key.
+        #
+        # @param [String] service_account_email The email address of the service
+        #   account. Used to create the HMAC key.
+        #
+        # @param [String] user_project If this parameter is set to a project ID
+        #   other than the current project, and that project is authorized for
+        #   the currently authenticated service account, transit costs will be
+        #   billed to the given project. The default is `nil`.
+        #
+        # @return [Google::Cloud::Storage::HmacKey]
+        #
+        def create_hmac_key service_account_email, user_project: nil
```

Comment:
@quartzmo could you make project_id an optional parameter? By default use project id instantiated at client creation.

Reply:
Yes, will do.

Reply:
Added `project_id` param to `Project#create_hmac_key`, `#hmac_key`, and `#hmac_keys`.